packages/eventstream-serde-universal/src/getChunkedStream.ts

```diff
@@ -0,0 +1,107 @@
+export function getChunkedStream(
+  source: AsyncIterable<Uint8Array>
+): AsyncIterable<Uint8Array> {
+  let currentMessageTotalLength = 0;
+  let currentMessagePendingLength = 0;
+  let currentMessage: Uint8Array | null = null;
+  let messageLengthBuffer: Uint8Array | null = null;
+  const allocateMessage = function (size: number) {
```

Comment:
arrow function
```suggestion
  const allocateMessage = (size: number) => {
```

Reply:
resolved in https://github.com/aws/aws-sdk-js-v3/pull/1082/commits/8e9bbc011160d4bf0fd0e799cf4873a875d95c12